import CustomException.CustomException;

public class Main {
    public static void main(String[] args) throws CustomException {
        System.out.println("Sistema iniciado!");
    }
}
